src/pal/inc/pal_mstypes.h

```diff
@@ -584,37 +584,29 @@ typedef LONG_PTR LPARAM;
 #define _PTRDIFF_T
 #endif
 
-#ifdef PAL_STDCPP_COMPAT
-
-#ifdef BIT64
-typedef unsigned long int uintptr_t;
-#else // !BIT64
-typedef unsigned int uintptr_t;
-#endif // !BIT64
-
-typedef char16_t WCHAR;
-
-#else // PAL_STDCPP_COMPAT
-
-typedef wchar_t WCHAR;
-#if defined(__linux__) 
+#ifndef PAL_STDCPP_COMPAT
```

Comment:
I would prefer making only the relevant change without refactoring. It doesn't seem that adding more ifdef blocks makes the code better. Could you please update your change in this file so that it just deletes the uintptr_t typedefs?

Reply:
Done.